ci/scripts/driver.sh

```diff
@@ -109,8 +114,8 @@ for pr in ${pr_list}; do
     for yaml_config in "${HOMEgfs}/ci/cases/"*.yaml; do
       pslot=$(basename "${yaml_config}" .yaml) || true
       export pslot
-      sed -i "/^base:/a\  ACCOUNT: \${SLURM_ACCOUNT}" "${pr_dir}/global-workflow/parm/config/gfs/yaml/defaults.yaml"
-      sed -i "/^base:/a\  ACCOUNT: \${SLURM_ACCOUNT}" "${pr_dir}/global-workflow/parm/config/gefs/yaml/defaults.yaml"
+      HOMEgfs_ci="${HOMEgfs}"
+      export HOMEgfs_ci
```

Comment:
sorry, but remind me the purpose of this?

Reply:
I need the full path to the yaml file to update the defaults to nems from the HOME ROOT of the repo where the CI code is .. in the driver script its HOMEgfs and I move it to the BASH env to HOMEgfs_ci and it gets resolved in the case yaml file [C96_atm3DVar.yaml](https://github.com/NOAA-EMC/global-workflow/blob/49983fab24c77470b82e7b5b8efc5cb1fb4e50f9/ci/cases/C96_atm3DVar.yaml)

Reply:
recall we can not use HOMEgfs in the BASH because that will crash setup_expt.py thus the _ci tag at the end.